src/py42/modules/securitydata.py

```diff
@@ -276,7 +273,8 @@ def stream_file_by_md5(self, checksum):
         """
         events = self._search_by_hash(checksum, MD5)
         if not len(events):
-            raise Py42ArchiveFileNotFoundError(checksum, "")
+            message = u"No files found with md5 checksum {}".format(checksum)
+            raise Py42Error(message)
```

Comment:
Would it be better to make a custom Exception class for this so we can easily catch it in certain circumstances?  Like a `Py42UnavailableFileError` that takes a hash type that was used to search for, and it's value

Reply:
`Py42HashUnavailableError`?

Reply:
Makes sense to me!

Reply:
`Py42HashNotFoundError` would mirror `Py42ArchiveFileNotFoundError`

Reply:
done!